frontend/src/lib/components/LemonSnack/LemonSnack.tsx

```diff
@@ -8,13 +8,22 @@ export interface LemonSnackProps {
     onClose?: () => void
     title?: string
     wrap?: boolean
+    className?: string
     'data-attr'?: string
+    color?: 'primary-extralight' | 'primary-highlight'
```

Comment:
This `primary-extralight` color was being used elsewhere, but it wasn't defined as an actual acceptable color. See: https://github.com/PostHog/posthog/pull/12877/files#diff-391b12e6ca682263152b5e6137a8933e7c6c4c894b003e365d4349a2f6e9c416L123-L125

Instead of having a color in the system that was defined as only being used for some special hover, but was actually used elsewhere, I decided to make it its own color. I agree that the naming is confusing. 

I'm not sure that I agree with using alpha variants, though, as the opacity will actually allow other things behind it to bleed through unless you are careful. In the below screenshot, the `Demo` Snack component and the `New Project` button use the same exact color `rgb(87, 199, 148, 0.2)`, but because the `Demo` Snack has another color behind it, it looks very different. (This obviously is not what this component will look like, just providing an example.)

![image](https://user-images.githubusercontent.com/18598166/204589320-ae2c97d7-465c-4f70-8737-62f4175e9dea.png)

Unless I am misunderstanding what you are suggesting?

Reply:
You are understanding it correctly. Indeed bleeding through would be the preferred behavior. Right now we need to maintain a bunch of extra hacks to make colors correctly in cases when the parent component is being hovered over/clicked, and this doesn't even work everywhere (e.g. info AlertMessages have a close button with no apparent hover state). This `primary-extralight` would be extra complexity on top of this. All solved by using translucent colors.
It is on the other hand very rare for perceptually transparent UI to be placed over a background and to want to ignore that background for hover states. I'll need to investigate this a bit more to be sure though.

Reply:
OK, i will merge this as-is and we can revisit the opacity question in another PR since it's not directly tied to this one.